docs/launch-behavior.md

```diff
@@ -0,0 +1,137 @@
+| `atom --add /b/3.md`        | [/a,/b 3.md]   | [/a 3.md] :new:    | [/a 3.md]    |
+| `atom --add /b`             | [/a,/b]        | [/a,/b]            | [/a,/b]      |
+| `atom --new-window /a/1.md` | [/a] [/a 1.md] | [/a] [ 1.md] :new: | [/a] [ 1.md] |
+| `atom --new-window /a`      | [/a] [/a]      | [/a] [/a]          | [/a] [/a]    |
+| `atom --new-window /b/3.md` | [/a] [/b 3.md] | [/a] [ 3.md] :new: | [/a] [ 3.md] |
+| `atom --new-window /b`      | [/a] [/b]      | [/a] [/b]          | [/a] [/b]    |
+
+### Open windows, one with a project root and one without
+
+> [/a] []
+
+| Action | Outcome (<=1.35.1) | Outcome (1.36.0) | Outcome (1.36.1) |
+| --- | --- | --- | --- |
+| `atom /a/1.md`              | [/a 1.md] []      | [/a 1.md] []          | [/a 1.md] []       |
+| `atom /a`                   | [/a] []           | [/a] []               | [/a] []            |
+| `atom /b/3.md`              | [/a] [/b 3.md]    | [/a] [] [ 3.md] :new: | [/a] [ 3.md] :new: |
+| `atom /b`                   | [/a] [/b]         | [/a] [] [/b]          | [/a] [] [/b]       |
+| `atom --add /a/1.md`        | [/a 1.md] []      | [/a 1.md] []          | [/a 1.md] []       |
+| `atom --add /a`             | [/a] []           | [/a] []               | [/a] []            |
+| `atom --add /b/3.md`        | [/a] [/b 3.md]    | [/a] [ 3.md] :new:    | [/a] [ 3.md]       |
+| `atom --add /b`             | [/a] [/b]         | [/a] [/b]             | [/a] [/b]          |
+| `atom --new-window /a/1.md` | [/a] [] [/a 1.md] | [/a] [] [ 1.md] :new: | [/a] [] [1.md]     |
+| `atom --new-window /a`      | [/a] [] [/a]      | [/a] [] [/a]          | [/a] [] [/a]       |
+| `atom --new-window /b/3.md` | [/a] [] [/b 3.md] | [/a] [] [ 3.md] :new: | [/a] [] [ 3.md]    |
+| `atom --new-window /b`      | [/a] [] [/b]      | [/a] [] [/b]          | [/a] [] [/b]       |
+
+## File manager integration actions
+
+### No open windows
+
+When no Atom windows are open, file manager context operations have the following outcomes:
+
+| Action | Outcome (<=1.35.1) | Outcome (1.36.0) | Outcome (1.36.1) |
+| --- | --- | --- | --- |
+| Click on `/a/1.md` | [/a 1.md] | [ 1.md] :new: | [ 1.md] |
+| Click on `/a`      | [/a]      | [/a]          | [/a]    |
+
+### Open window, no project roots
+
+With the following starting state:
+
+> []
+
+| Action | Outcome (<=1.35.1) | Outcome (1.36.0) | Outcome (1.36.1) |
+| --- | --- | --- | --- |
+| Click on `/a/1.md` | [/a 1.md] | [] [ 1.md] :new: | [ 1.md] :new: |
+| Click on `/a`      | [/a]      | [] [/a]          | [] [/a]       |
+
+### Open window, project root
+
+With the following starting state:
+
+> [/a]
+
+| Action | Outcome (<=1.35.1) | Outcome (1.36.0) | Outcome (1.36.1) |
+| --- | --- | --- | --- |
+| Click on `/a/1.md` | [/a 1.md]      | [/a 1.md]          | [/a 1.md]    |
+| Click on `/a`      | [/a]           | [/a]               | [/a]         |
+| Click on `/b/3.md` | [/a 3.md]      | [/a] [ 3.md] :new: | [/a] [ 3.md] |
```

Comment:
I think 1.36.1 should restore the previous behavior. This change in behavior wasn't an explicit goal of #18608 (AFAIK), and since launching a new window is slow, this change arguably degrades the user experience.

```suggestion
| Click on `/b/3.md` | [/a 3.md]      | [/a] [ 3.md] :new: | [/a 3.md] |
```